keps/sig-network/2091-admin-network-policy/README.md

```diff
@@ -408,231 +418,418 @@ also like to consider the following set of proposals as future work items:
 ### AdminNetworkPolicy API Design
 
 The following new `AdminNetworkPolicy` API will be added to the `policy.networking.k8s.io`
-API group.  
+API group:
```

Comment:
I'm not sure what the right thing to do here is... we don't want to have to update the KEP every time we change the API.

I think it might be better to just say that this was the originally proposed API, and then include a link to the current alpha API. (ie a link to a specific git hash of the API definition). And then we can update again for beta and GA.

Reply:
yeah, we don't HAVE to do this.  It's understood that API changes happen.  We might update it all when we want to move to beta.